plenoptic/metric/perceptual_distance.py

```diff
@@ -350,14 +363,17 @@ def downsample(img):
 
 
 def normalized_laplacian_pyramid(im):
```

Comment:
Do we want to allow for user-specified number of scales?

Reply:
The normalization weights are learned for the Laplacian pyramid with 6 scales, so it has to be 6 scales.

Reply:
Sounds good